eve/main.yml

```diff
@@ -221,9 +222,10 @@ stages:
           command: >
             ssh -F bootstrap_ssh_config bootstrap
             sudo mkdir -p /etc/metalk8s &&
-            scp -F bootstrap_ssh_config bootstrap.yaml bootstrap: &&
+            scp -F bootstrap_ssh_config bootstrap.yaml.in bootstrap: &&
```

Comment:
How about replacing this with something along the lines of

```
sed "s/@VERSION@/%(prop:metalk8s_short_version)s/g" bootstrap.yaml.in | ssh -F bootstrap_ssh_config bootstrap sudo tee /etc/metalk8s/bootstrap.yaml
```

Reply:
It's not needed however, since we're not configuring the mountpoint in this file anymore